NOTES-NONSTOP.md

```diff
@@ -44,6 +44,19 @@ instead of `nsx` in the set above.
 You cannot build for TNS/E for FIPS, so you must specify the `no-fips`
 option to `./Configure`.
 
+Linking and Loading Considerations
+----------------------------------
+
+Because of how the NonStop Common Runtime Environment (CRE) works, there are
+restrictions on how programs can link and load with OpenSSL libraries.
+On current NonStop platforms, programs cannot both statically link OpenSSL
+libraries and dynamically load OpenSSL DLLs concurrently. If this is done,
+there is a high probability of encountering a SIGSEGV condition relating to
+`atexit()` processing when the DLL is unloaded and when the program terminates.
+This limitation applies to all OpenSSL DLL components.
```

Comment:
I'd replace DLL with `shared library` as the DLL is kind of Windows-specific acronym.

Reply:
Will make that change imminently.